client/extensions/woocommerce/store-sidebar/store-ground-control.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+const Gridicon = require( 'gridicons' ),
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/13837/commits/5213d1766e95f25588cf319dc7a1b8f1f2c48e9a